### ISSUE TITLE
Source code not found on torchvision documentation page.

### ISSUE THREAD

**Issue by nateanl** [pytorch/vision#4814]
### 📚 The doc issue

When I click the ``[Source]`` link of the [ImageNet](https://pytorch.org/vision/stable/datasets.html#imagenet) dataset, the webpage shows ``404 File not found``.

![image](https://user-images.githubusercontent.com/8653221/139680634-5f92db6d-e2ad-4aa9-b7e8-22edaca2246f.png)
I checked the link for other methods, such as ``torchvision.io``. It seems all links are not valid.

cc @NicolasHug @prabhat00155 @fmassa

**Comment by NicolasHug** [comment]
Thanks for the report @nateanl 

 Comparing https://github.com/pytorch/vision/tree/gh-pages/0.11/_modules and https://github.com/pytorch/vision/tree/gh-pages/0.10/_modules, it looks like an entire folder is missing

@mattip Do you know what could have gone wrong here?

**Comment by vadimkantorov** [comment]
Related: https://github.com/pytorch/pytorch/issues/67237

**Comment by mattip** [comment]
Not sure. It is only in the `0.11` directory, in `main` (run nightly) the subdirectory exists. I will make a PR to restore them, although I really don't understand how this happened.

**Comment by mattip** [comment]
The problem is [this `torchvision/` directive](https://github.com/pytorch/vision/blob/f1e783e9a29a27507735a723a06ee205c1cf5c32/.gitignore#L7) in the .gitignore file on the gh-pages branch.

**Comment by mattip** [comment]
When I try to rebuild the 0.11.0 docs, I get this sphinx error and docs are not created. Is that known?
```
Warning, treated as error:
/home/matti/miniconda3/envs/throwaway/lib/python3.7/site-packages/torchvision/ops/deform_conv.py 
    :docstring of torchvision.ops.deform_conv.DeformConv2d.forward:7:
    Field list ends without a blank line; unexpected unindent.
make: *** [Makefile:41: html] Error 2

```

**Comment by mattip** [comment]
> When I try to rebuild the 0.11.0 docs, ...

Nvrmind, I had cruft from a levtover build in my `docs/source` directory

**Comment by nateanl** [comment]
Resolved by #4825. Thanks @mattip for working on this!
